poms/bnd/pom.xml

```diff
@@ -59,6 +59,8 @@
     <karaf.version>4.2.2</karaf.version>
     <sat.version>0.6.1</sat.version>
     <slf4j.version>1.7.21</slf4j.version>
+    <jaxb.version>1.0.0</jaxb.version>
```

Comment:
Don't use it.
If we create a BOM for JAXB and JAX-WS the version of that POM will be the openHAB version.
As long as addon and core (or the place we will place that stuff) does not use different versions it will be `${project.version}` for the dependency declaration (otherwise e.g. `${ohc.version}`).